libraries/chain/worker_evaluators.cpp

```diff
@@ -470,17 +466,15 @@ namespace golos { namespace chain {
         }
 
         if (wpao_itr != wpao_idx.end()) {
-            _db.modify(*wpao_itr, [&](worker_payment_approve_object& wpao) {
-                wpao.state = o.state;
-            });
-        } else {
-            _db.create<worker_payment_approve_object>([&](worker_payment_approve_object& wpao) {
-                wpao.approver = o.approver;
-                wpao.post = wro_post.id;
-                wpao.state = o.state;
-            });
+            return;
```

Comment:
Here should be error - operation shouldn't be included into block.

Reply:
@afalaleev https://github.com/GolosChain/golos/issues/1139